tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -292,4 +283,46 @@ spec:
 			helper.MatchAllInOutput(jsonOut, []string{msg, msgWithQuote})
 		})
 	})
+
+	Context("When deleting Operator backed service", func() {
```

Comment:
I can see this spec caries the same steps https://github.com/openshift/odo/pull/3564/files#diff-49f09fb5341717455c0421b3c592e685R72 plus a validation steps

```
// now try deleting the same service again. It should fail with error message
stdOut := helper.CmdShouldFail("odo", "service", "delete", "EtcdCluster/example", "-f")
Expect(stdOut).To(ContainSubstring("Couldn't find service named"))
```
To improve the test time consumption, IMO you can just update the existing [spec](https://github.com/openshift/odo/pull/3564/files#diff-49f09fb5341717455c0421b3c592e685R72) along with the above validation step. For this may be you need to update the ```Context``` text and ```It``` message.

Reply:
Combined the two. Great tip. Thanks! :+1: